NB01/README.md

```diff
@@ -60,14 +61,14 @@ You should see a symbol (*) next to the `base` environment:
 
 ![](conda_env.png)
 
-> Note: If you already know how to work with Anaconda environments, do not forget switching to the `base` by running `source activate base` command. 
+> Note: If you already know how to work with Anaconda environments, do not forget switching to the `base` by running `conda activate base` command.
```

Comment:
I think that by default, any new terminal opens  the base environment. I'm not 100% sure, but I think if you activate another env while inside and env (non base), some funny stuff can happen (I think it's happened to me more than once). Instead of activating `base`, I would recommend doing `conda deactivate` *until* you arrive at `base`.